### ISSUE TITLE
Strip tags from nav item titles

### ISSUE THREAD

**Issue by wilhelmer** [squidfunk/mkdocs-material#1072]
For better readability, I have added manual line breaks to some of my topic titles in `mkdocs.yml`. Example:
```
- Software Installation <br>(Windows 10): software-installation-(windows-10).md
```
This is rendered as follows:
``` html
<a href="software-installation-(windows-10).html" title="Software Installation <br>(Windows 10)" class="md-nav__link">
      Software Installation <br>(Windows 10)
</a>
```
I think the `<br>` tag shouldn't be included in the title. I had some issues with that when reading the title in my custom JS files.

**Suggestion**

Add `striptags` to the item titles in nav-item.html:
``` html
<a href="{{ nav_item.url | url }}" title="{{ nav_item.title | striptags }}" class="md-nav__link">
  {{ nav_item.title }}
</a>
```

**Comment by squidfunk** [comment]
Thanks for reporting. Do you have a screenshot how it looks? Also, is `striptags` generally available in Jinja? Just for a quick comparison.

**Comment by wilhelmer** [comment]
**Screenshot**
<img width="269" alt="nav-item-tags" src="https://user-images.githubusercontent.com/2234833/56157675-5d964700-5fc0-11e9-8bb0-52b83072392c.png">

**"striptags" in the official Jinja documentation**
http://jinja.pocoo.org/docs/2.10/templates/#filters
http://jinja.pocoo.org/docs/2.10/templates/#striptags

**Comment by squidfunk** [comment]
Perfect, care to submit a PR? I can merge it directly and prepare a release.

**Comment by squidfunk** [comment]
Released as part of 4.1.2.
